.mailmap

```diff
@@ -2,8 +2,8 @@ Anisha Keshavan <akeshavan@gmail.com> <akeshavan@ucla.edu>
 Anisha Keshavan <akeshavan@gmail.com> <keshavan@berkeley.edu>
 Ariel Rokem <arokem@berkeley.edu>
 Ariel Rokem <arokem@berkeley.edu> <arokem@gmail.com>
-Derek Pisner <dpisner@utexas.edu>
-Derek Pisner <dpisner@utexas.edu> dPys <dpisner@utexas.edu>
+Derek Pisner <dpysalexander@gmail.com>
+Derek Pisner <dpysalexander@gmail.com> dPys <dpysalexander@gmail.com>
```

Comment:
This will remove your name from old commits done with the old email.

That said, please send a separate PR with this edit.
```suggestion
Derek Pisner <dpysalexander@gmail.com> dPys <dpysalexander@gmail.com>
Derek Pisner <dpysalexander@gmail.com> <dpisner@utexas.edu> 
```

Reply:
See https://github.com/nipreps/eddymotion/pull/119